lib/src/base_request.dart

```diff
@@ -6,6 +6,8 @@ import 'dart:collection';
 
 import 'package:meta/meta.dart';
 
+import '../http.dart';
```

Comment:
can we do something more narrow?

Reply:
A bit. Sadly, `get` is in the root lib. Added a `show` so it's clear.